### PR TITLE
Remove wrong statement on HOFs in Python

### PR DESCRIPTION
**Pull request by tuffnatty**
See the discussion on https://github.com/dry-python/functional-jargon-python/commit/926a6f06bc134d39f8008dbbe16e63fccff9da6a#r43380159

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +205,6 @@ There is a little difference between __arguments__ and __parameters__:
 ## Higher-Order Functions (HOF)
 
 A function that takes a function as an argument and/or returns a function, basically we can treat functions as a value.
-In Python every function/method is a Higher-Order Function.
```
Comment (thepabloaguilar) [inline README.md:208]: This is not the way, you don't need to remove the line!!

Beyond functional jargon, it's functional jargon __using__ Python. So, this information is important!
Just reformulating the phrase should be enough.

**Review by thepabloaguilar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tuffnatty) [reply README.md:208]: Well, it's your text, I am not sure what you were trying to say, but the current wording is false.
Maybe "Python supports higher-order functions."?

Comment (thepabloaguilar) [reply README.md:208]: Sorry for the long delay!!
What about, "In Python every function/method can be a Higher-Order Function"??

Comment (sobolevn) [reply README.md:208]: 👍

**Pull request closed without merging**